ui/app/src/components/PluginManagement/PluginManagement.tsx

```diff
@@ -53,8 +53,7 @@ import LookupTable from '../../models/LookupTable';
 import GlobalState from '../../models/GlobalState';
 import GlobalAppToolbar from '../GlobalAppToolbar';
 import { useSelection } from '../../hooks/useSelection';
-import { useActiveSiteId } from '../../hooks/useActiveSiteId';
-import { useMount } from '../../hooks/useMount';
+import { useActiveSiteId, useEnhancedDialogState, useMount, useWithPendingChangesCloseRequest } from '../../hooks';
```

Comment:
Rollback. Leave hooks imported individually. Configure your IDE not to fix these or pull so you don't get the warning